mslib/msui/mscolab_version_history.py

```diff
@@ -108,9 +108,12 @@ def load_current_waypoints(self):
         }
         url = url_join(self.mscolab_server_url, 'get_project_by_id')
         res = requests.get(url, data=data)
-        xml_content = json.loads(res.text)["content"]
-        waypoint_model = WaypointsTableModel(name="Current Waypoints", xml_content=xml_content)
-        self.currentWaypointsTable.setModel(waypoint_model)
+        if res.text != "False":
+            xml_content = json.loads(res.text)["content"]
+            waypoint_model = WaypointsTableModel(name="Current Waypoints", xml_content=xml_content)
+            self.currentWaypointsTable.setModel(waypoint_model)
+        else:
+            show_popup(self, "Error", "Some error occurred while fetching data!")
```

Comment:
see... here an action is taken for the False case.

Reply:
this needs to become reviewed with #975